examples/durable/DurableApp/MonitorOrchestrator/run.ps1

```diff
@@ -0,0 +1,24 @@
+param($Context)
+
+Write-Host 'MonitorOrchestrator: started.'
+
+$jobId = $Context.Input.JobId
+$pollingInterval = $Context.Input.PollingInterval
+$expiryTime = $Context.Input.ExpiryTime
+
+while ($Context.CurrentUtcDateTime -lt $expiryTime) {
+    $jobStatus = Invoke-ActivityFunction -FunctionName 'GetJobStatus' -Input $jobId
+    if ($jobStatus -eq "Completed") {
+        # Perform an action when a condition is met.
+        $output = Invoke-ActivityFunction -FunctionName 'SendAlert' -Input "foo"
```

Comment:
Let's come up with a more meaningful and realistic sample value than "foo". Perhaps passing $jobId there makes sense?

Reply:
$jobId makes sense, but I believe $machineId might be better to maintain parallelism with the C#, JavaScript, and Python templates.

Reply:
The first priority here is to make it a working self-sufficient sample. If you want to pass `$machineId`, it should be initialized first, otherwise `$null` will be passed. Or pass a hardcoded value like `"machine id"`. `"foo"` is just strange: as a reader of this code, I have no idea how to interpret it.

Making this code consistent with samples for other languages is nice but completely optional here. This consistency would be important on a documentation page where multiple samples are presented together on different tab, but this is not where we are.